### DISCUSSION TITLE
Toggle button for privacy banner

### DISCUSSION THREAD

**Discussion by lakshaa** [globaleaks/globaleaks-whistleblowing-software#3734] [Q&A] [answered]
I can enable the privacy banner for Tor to be always visible. However, I wish to display the little toggle button so that users can hide the banner at their own convenience. I remember it was working previously, then we migrated the website to another server and now it does not work. Pls help, thank you.

**Comment by evilaliv3** [comment, marked as answer]
Thank you. For security reasons we have actually decided to always show this important reminder to users.
